backend/TreeOfAKind.Infrastructure/Migrations/20201204184823_AddPeopleToTree.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TreeOfAKind.Infrastructure.Migrations
+{
+    public partial class AddPeopleToTree : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Biography",
+                schema: "trees",
+                table: "People",
+                type: "nvarchar(max)",
+                maxLength: 10000,
```

Comment:
I assume there's no difference for sql server between "nvarchar(max)" with max length constraint and "nvarchar(10000)"

Reply:
There is, you should keep sane maxes in respected projects.